tests/functional/strategy/test_misc.py

```diff
@@ -134,3 +134,55 @@ def test_set_metadataURI(gov, strategy, rando):
     assert strategy.metadataURI() == "ipfs://test2"
     with brownie.reverts():
         strategy.setMetadataURI("ipfs://fake", {"from": rando})
+
+
+def test_sandwich_attack(
+    chain, TestStrategy, web3, token, gov, vault, strategist, rando
+):
```

Comment:
Seems like this test show live under `functional/vault/`

Reply:
done in latest